ledger-core/v2/conveyor/sm_pulse_slot.go

```diff
@@ -212,7 +212,7 @@ func (p *PulseSlotMachine) stepPresentLoop(ctx smachine.ExecutionContext) smachi
 	case !nextPollTime.IsZero():
 		return ctx.WaitAnyUntil(nextPollTime).ThenRepeat()
 	case !p.innerMachine.IsEmpty():
```

Comment:
this case could be removed because it duplicates the code after switch

Reply:
done

Reply:
hmm, am I missing something? or you've forgot to push?